src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,53 @@
-export const Tabs = () => {};
+import { FC } from 'react';
+import cn from 'classnames';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string;
+  onTabSelected: (tab: Tab) => void,
+};
+
+export const Tabs: FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTabs = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map((tab) => {
+            return (
+              <li
+                className={cn(
+                  { 'is-active': tab.id === selectedTabs.id },
```

Comment:
move it to the variable
```suggestion
                  { 'is-active': ... },
```